src/Tests/ApprovalFiles/TemplateTests.ScAdapterServiceDiffFramework.approved.txt

```diff
@@ -11,14 +11,11 @@ namespace ScAdapterServiceDiffFramework
 {
     class AdapterHostedService : IHostedService
     {
-        // TODO: give the adapter an appropriate name
```

Comment:
Should we at least bring back the TODOs into the configs everywhere? Apparently previous people working on this code base felt it is important to highlight with a TODO.

Reply:
Reason: This class is already not really annotated with TODOs so having one less is not a problem